styles.css

```diff
@@ -1,10 +1,10 @@
 #img-div {
-  width: 100%;
-  text-align: center;
-}
-
-#image {
-  display: block;
-  margin: auto;
-  width: 15%;
-}
+    width: 100%;
+    text-align: center;
+  }
+  
+  #image {
+    display: block;
+    margin: auto;
+    width: 15%;
+  }
```

Comment:
Please, your file shouldn't have newlines after the code. It makes your file larger than required.